web-app/README.md

```diff
@@ -0,0 +1,28 @@
+# TelemetryView
```

Comment:
kk, I will convert the readme files tomorrow. Never heard of asciidoc before so I will have to read up on that format.